integration/docker/entrypoint.sh

```diff
@@ -43,16 +43,25 @@ alluxio_env_vars=(
   ALLUXIO_WORKER_JAVA_OPTS
 )
 
+# Map of environment variables that violates the "_ to ."-rule
+declare -A alluxio_env_violators=(
+    ["ALLUXIO_UNDERFS_S3A_INHERIT_ACL"]="alluxio.underfs.s3a.inherit_acl"
```

Comment:
In the jira ticket, it also mentions `alluxio.master.format.file_prefix`. Could you add that too?